django/journal/api.py

```diff
@@ -0,0 +1,38 @@
+from . import models, serializers
+from rest_framework import viewsets, permissions
+
+
+class ProjectViewSet(viewsets.ModelViewSet):
+    """
+    Creates a ViewSet for managing CRUD operations on the Project model
+    """
+
+    queryset = models.Project.objects.all()
```

Comment:
The models all have a `admin_published` field. Should the api be restricting to showing only when this is true?

Reply:
Good spot, thanks. Have fixed this and tested it's working as expected.